src/ml-search.service.js

```diff
@@ -1137,16 +1141,16 @@
       }
 
       params = params || {};
-      params.start = params.start || 1;
-      params.limit = params.limit || 20;
+      params.start = params.start !== undefined ? params.start : 1;
+      params.limit = params.limit !== undefined ? params.limit : 20;
```

Comment:
Yes exactly. The getAggregates passes through limit=0 to only get aggregates, and not bother about values. My test case was complaining about it.. ;-)